src/components/quiz/QuizPage.tsx

```diff
@@ -2,18 +2,34 @@ import React, { useState } from "react";
 import { QuizQuestions } from "./QuizQuestions";
 import { UserDetailsForm } from "./UserDetailsForm";
 import { questions } from "./questions";
+import { Reward } from "./Reward";
+import './QuizQuestions.scss';
```

Comment:
Prefer double quotes
```suggestion
import "./QuizQuestions.scss";
```